src/widgets.jl

```diff
@@ -155,3 +155,29 @@ function jsrender(session::Session, slider::RangeSlider)
     onjs(session, style, create_slider)
     return rangediv
 end
+
+
+struct Checkbox <: JSServe.AbstractWidget{Bool}
+    value::Observable{Bool}
+    attributes::Dict{Symbol, Any}
+end
+
+function Checkbox(value::Bool; kw...)
+    return Checkbox(Observable(value), Dict{Symbol, Any}(kw))
+end
+
+function JSServe.jsrender(tb::Checkbox)
+    # unchecked in css is encoded by leaving out the attribute (*sigh*)
```

Comment:
I agree with the "sigh", but this is actually an HTML thing, and HTML5 adds their own bizarre restrictions to try and make things less counterintuitive, see [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#Boolean_Attributes).